mesop/events/events.py

```diff
@@ -13,9 +13,10 @@ class ClickEvent(MesopEvent):
 
   Attributes:
       key (str): key of the component that emitted this event.
+      is_target (bool): Whether the clicked target contains the event handler.
```

Comment:
Nit: it's more like the clicked target is the *component* which attached the event handler. (Technically an event handler can be attached to multiple components)